spec/pandadoc/api/document_spec.rb

```diff
@@ -65,6 +65,14 @@
 
       expect(client_spy).to have_received(:post_json).with('/documents', token, params)
     end
+
+    it 'allows images as params' do
```

Comment:
This is really similar to the test above about `tags` and it doesn't seem to really be testing much but I added it anyway 😒